projects/mac_ai/testing/prepare_mac_ai.py

```
@@ -5,8 +5,8 @@
 from projects.core.library import env, config, run, configure_logging, export
 from projects.matrix_benchmarking.library import visualize
 
-import prepare_llama_cpp, utils, remote_access, podman_machine, brew, podman, prepare_virglrenderer
-import llama_cpp, ollama, ramalama
+import utils, remote_access, podman_machine, brew, podman, prepare_virglrenderer, prepare_release
```

Flake8 (7.2.0)

[error] multiple imports on one line

(E401)